plugins/module_utils/_stormssh.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is based on
+# the config parser from here: https://github.com/emre/storm/blob/master/storm/parsers/ssh_config_parser.py
+# Copyright (C) <2013> <Emre Yilmaz>
+# SPDX-License-Identifier: MIT
+
+import os
+import re
+import traceback
+from operator import itemgetter
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.basic import missing_required_lib
+
+try:
+    import paramiko
```

Comment:
Better do
```suggestion
    from paramiko.config import SSHConfig
```
here and `SSHConfig = object` in the `except ImportError` branch, so that this file can also be imported when `paramiko` cannot be imported.